README-install-guide.md

```diff
@@ -16,7 +16,7 @@ helm upgrade --install kubecost \
  --namespace kubecost --create-namespace \
  -f https://raw.githubusercontent.com/kubecost/openshift-helm-chart/1.98.0-rc.4.1/cost-analyzer/disable-psps.yaml \
```

Comment:
@jessegoodier I think we should combine disable-psps.yaml, values-restricted-podsecurity.yaml and kubecost-primary-cluster-settings.yaml into 1 values file such as values-openshift.yaml to simplify the installation command and align with what we have in previous version.